runtimes/eoapi/raster/eoapi/raster/app.py

```diff
@@ -7,6 +7,7 @@
 
 import jinja2
 import pystac
+from eoapi import auth
```

Comment:
I think it's better to avoid `importing` a full submodule (especially because eoapi is not a module itself) 

As mentioned in my PR, I usually try to avoid importing a full module if it's only for couple methods or class 

```python
from eoapi.auth import AuthSettings, OpenIdConnectAuth
from .config import ApiSettings
from .logs import init_logging
```

Reply:
Why do you prefer to avoid importing an entire module? I find it easier to both write and track where things come from.

Reply:
mostly just an habit I think

At least for `auth` I think it will be safer